frontend-shared/src/index.js

```diff
@@ -1,3 +1,8 @@
-module.exports = {
-  ...require('./svg-icon'),
-};
+// Components
+export { SvgIcon, registerIcons } from './svg-icon';
```

Comment:
I switched to explicitly listing the exports here rather than exporting everything because that makes it easier to see what is included in the package by looking at just this file. It also makes breaking changes to the package's API due to adding, removing or renaming symbols more obvious.

Reply:
Makes sense